### PR TITLE
Clean up mesh include code and comments

### PR DESCRIPTION
**Pull request by aaronfranke**
This PR does some misc cleanup related to mesh and 3D resource code.

* Ensure several `#ifndef _3D_DISABLED` statements have their `#endif` statements with a comment.
* Only register sky materials when 3D is enabled, since sky materials are only for 3D (AFAIK).
* Use some more explicit paths in includes.
* Remove unused `#include <stdlib.h>`.
* Remove some unhelpful comments.

### CODE REVIEW

File: scene/animation/animation_player.cpp [hunk 1]
```diff
@@ -375,7 +375,7 @@ void AnimationPlayer::_ensure_node_caches(AnimationData *p_anim, Node *p_root_ov
 					node_cache->init_rot = rest.basis.get_rotation_quaternion();
 					node_cache->init_scale = rest.basis.get_scale();
 				} else {
-					// no property, just use spatialnode
+					// no property, just use node_3d
```
Comment (fire) [inline scene/animation/animation_player.cpp:378]: We're not allowed to use "just" in a our style guide I think.

**Review by fire — APPROVED** [review]
Style changes looks good. Only one change requested.

**Comment by Calinou** [comment]
> Only register sky materials when 3D is enabled, since sky materials are only for 3D (AFAIK).

This is indeed the case – like FogMaterial.

**Review by clayjohn — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**